Comments/components/commentBrowser.js

```diff
@@ -65,49 +78,63 @@ export class CommentBrowser extends Window {
       clipMode: 'auto',
       name: 'comment container'
     });
-    this.containerLayout = new Morph({
+    this.layoutContainer = new Morph({
       layout: new VerticalLayout({
         spacing: 5,
         orderByIndex: true
       }),
       name: 'comment container layout'
     });
-    this.container.addMorph(this.containerLayout);
+    this.container.addMorph(this.layoutContainer);
     this.addMorph(this.container);
   }
 
   close () {
-    super.close();
+    // super.close();
     const topbar = $world.getSubmorphNamed('lively top bar');
     if (topbar) {
       topbar.uncolorCommentBrowserButton();
     }
-    this.removeCommentIndicators();
-    instance = undefined;
+    this.remove();
   }
 
-  async refreshCommentGroupMorphs () {
+  async initializeCommentGroupMorphs () {
     const commentGroupMorphs = [];
     await Promise.all($world.withAllSubmorphsDo(async (morph) => {
       if (morph.comments.length == 0) {
         return;
       }
+      morph.comments.forEach(async (comment) => {
+        await this.addCommentForMorph(comment, morph);
+      });
+    }));
+  }
+
+  async addCommentForMorph (comment, morph) {
+    if (morph.id in this.commentGroups) {
+      await this.commentGroups[morph.id].addCommentMorph(comment);
+      this.commentGroups[morph.id].relayout();
```

Comment:
Warum geschieht das nicht direkt in `addCommentMorph`?

Reply:
the relayout line will be removed

Reply:
Worauf beziehst du dich genau? Das relayout ist rausgeflogen, das gibt es als Methode nicht. Oder meintest du was anderes?